R/pkg/R/DataFrame.R

```diff
@@ -3058,7 +3057,7 @@ setMethod("str",
 #' @note drop since 2.0.0
 setMethod("drop",
           signature(x = "SparkDataFrame"),
-          function(x, col, ...) {
```

Comment:
This actually follows from the discussion in #14705. A summary may be seen at https://github.com/apache/spark/pull/14735#discussion_r75661714